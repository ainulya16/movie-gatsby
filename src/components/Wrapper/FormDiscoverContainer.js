import styled from 'styled-components';

const FormDiscoverContainer = styled.div`
  /* padding: 0 50px; */
  margin: 48px 0px 24px 0px;
`;

export default FormDiscoverContainer;
