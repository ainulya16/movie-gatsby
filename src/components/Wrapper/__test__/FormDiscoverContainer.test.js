import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import FormDiscoverContainer from '../FormDiscoverContainer';

describe('FormDiscoverContainer', () => {
  it('should be exists', () => {
    const wrapper = shallow(<FormDiscoverContainer />);
    expect(wrapper.exists()).toBe(true);
  });
  it('should match the snapshot', () => {
    const tree = renderer.create(<FormDiscoverContainer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
