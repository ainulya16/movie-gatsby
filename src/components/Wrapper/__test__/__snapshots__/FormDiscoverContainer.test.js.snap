// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormDiscoverContainer should match the snapshot 1`] = `
.c0 {
  margin: 48px 0px 24px 0px;
}

<div
  className="c0"
/>
`;
